telemetry/statsd/src/test/scala/io/buoyant/telemetry/statsd/MetricTest.scala

```diff
@@ -0,0 +1,98 @@
+package io.buoyant.telemetry.statsd
+
+import com.timgroup.statsd.StatsDClient
+import org.scalatest._
+
+class MetricTest extends FunSuite {
+
+  class MockStatsDClient extends StatsDClient {
+
+    var lastName = ""
+    var lastValue = ""
+
+    def count(x$1: String, x$2: Long, x$3: Double, x$4: String*): Unit = {
```

Comment:
were these variable names autogenerated by something?  that's probably fine for the unimplemented methods but for the ones where we provide implementations here it might be helpful to have real variable names.